src/components/spreadsheet/config/common-column-definitions.ts

```diff
@@ -169,3 +174,39 @@ export const booleanColumnDefinition = (displayName: string, tab: string): ColDe
         },
     };
 };
+
+export const rowIndexColumnDefinition = (tabUuid: string): ColDef => {
+    return {
+        colId: 'rowIndex',
+        headerName: '',
+        cellRenderer: (params: any) => {
+            // For pinned rows, use the RowIndexCellRenderer which handles the calculate icon
+            if (params.node.rowPinned) {
+                return RowIndexCellRenderer(params);
+            }
+
+            if (!params.node) {
+                return null;
+            }
+
+            // Get the actual displayed row index
+            const displayedRowIndex = params.api.getDisplayedRowCount() > 0 ? params.node.rowIndex : null;
+
+            // Add 1 to convert from 0-based to 1-based indexing
+            return displayedRowIndex !== null ? displayedRowIndex + 1 : null;
```

Comment:
```suggestion
            return null;
```
What's the point of having these inedexes ?